src/binary/Y2RubyComponent.cc

```diff
@@ -75,16 +76,30 @@ Y2Namespace *Y2RubyComponent::import (const char* name)
   args->add (YCPString(/*module*/ name));
   args->add (YCPString(/*module*/ module));
 
-  try {
-    YRuby::loadModule (args);
+  if (YRuby::loadModule (args))
+  {
     y2debug("Module '%s' loaded", name);
     // introspect, create data structures for the interpreter
     Y2Namespace * res = new YRubyNamespace (name);
     namespaces[name] = res;
     return res;
-  } catch (exception& e) {
```

Comment:
well, in fact it cannot throw, this catch is never called because:

1. ruby is pure C, so no exceptions
2. YNamespace do not throw

so I think it is just old bogus code and I do not expect anything to be raised here.